npm-react/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "react",
   "description": "React is a JavaScript library for building user interfaces.",
-  "version": "0.13.1",
+  "version": "0.13.2-alpha",
```

Comment:
It is for my branch, the last commit will be the one that bumps to 0.13.2 and then that gets tagged.